ksim/rewards.py

```diff
@@ -395,6 +396,16 @@ def joint_threshold_validator(
         raise ValueError(f"Joint threshold must be a float array, got {arr.dtype}")
 
 
+@attrs.define(frozen=True, kw_only=True)
+class GeomContactPenalty(Reward):
+    """Penalty for any contact forces on the geoms."""
+
+    observation_name: str = attrs.field()
+
+    def __call__(self, trajectory: Trajectory) -> Array:
+        return trajectory.obs[self.observation_name].mean(axis=-1)
```

Comment:
i would probably just rename this as something like `ObservationMeanPenalty` and let it be used for general observations, not just geom contact